dvl_gazebo/launch/teledyne_whn_standalone.launch

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0"?>
+<launch>
+  <arg name="gui" default="true"/>
+  <arg name="paused" default="false"/>
+  <arg name="world_name" default="$(find dvl_gazebo)/worlds/uuv_dave_ocean_waves_nogravity.world"/>
+  <arg name="set_timeout" default="false"/>
+  <arg name="timeout" default="0.0"/>
+ 
+  <!-- use Gazebo's empty_world.launch with uuv_dave_ocean_waves.world -->
+  <include file="$(find gazebo_ros)/launch/empty_world.launch">
+      <arg name="world_name" value="$(arg world_name)"/>
+      <arg name="paused" value="$(arg paused)"/>
+      <arg name="use_sim_time" value="true"/>
+      <arg name="gui" value="$(arg gui)"/>
+      <arg name="headless" value="false"/>
+      <arg name="debug" value="false"/>
+      <arg name="verbose" value="false"/>
+  </include>
+
+  <!-- Load and spawn the model, starts the joint & state publishers, make the model move -->
+  <param name="teledyne_whn" textfile="$(find dvl_gazebo)/models/teledyne_whn_urdf/model.urdf"/>
+  <node name="spawn_sonar" pkg="gazebo_ros" type="spawn_model" 
+        args="-z -5.0 -P 1.5707963267948966 -urdf -param teledyne_whn -model teledyne_whn"
+        respawn="false" output="screen" />
+  <node name="joint_state_publisher" pkg="joint_state_publisher" type="joint_state_publisher" respawn="false">
```

Comment:
How come you need the joint and robot state publishers?  Are you using rviz - or does the plugin require the transforms?

Reply:
The DVL plugin needs the transforms to map the ray sensors to the DVL frame.  It doesn't actually use them to derive the DVL solution, but it kicks out of the loop without publishing the DVL solution if the transforms aren't available.

Reply:
I'll take a look at the error--I know what it is and thought that I had fixed it before the commit (just needs a remap).

For the tutorial, I'll write up a quick README for the package for now and put it into the wiki when I get a chance.